project/src/Company.cpp

```diff
@@ -0,0 +1,10 @@
+ class Company{
+        public : 
+        string name;
+        void input( string n){
```

Comment:
remove functions **input()** and **display** 

Instead add Getters and Setters for name. Read this for more information: 

https://www.w3schools.com/cpp/cpp_encapsulation.asp